packages/backend/src/app/flows/flow/flow.service.ts

```diff
@@ -141,4 +152,18 @@ export const flowService = {
         await flowInstanceService.onFlowDelete({ projectId, flowId })
         await flowRepo.delete({ projectId: projectId, id: flowId })
     },
+    async count(req:{
+        projectId:string
+        folderId?:string
+        allFlows:boolean
+    }) :Promise<number>{
+        if(req.allFlows) {
+            return flowRepo.count()
+        }
+        return flowRepo.count({
+            where:[{folderId:req.folderId, projectId:req.projectId}],
```

Comment:
how does this support the case when folderId is null?

Reply:
this is fixed in another branch, I started passing null when folderId is undefined .